### PR TITLE
Update glocaltokens to 0.3.0

### LINKED ISSUE #160: Update glocaltokens to v0.3
@DurgNomis-drol, @KapJI as promised here is a new version of `glocaltokens`
See https://github.com/leikoilja/glocaltokens/releases/tag/v0.3.0 (See the breaking changes)

Before we upgrade `glocaltokens` we need to make sure it's not going to break the integration.

### PR DESCRIPTION
**Pull request by KapJI**
- It has improved `gpsoauth` which should make authorisation more reliable
- And improved logging which should properly redact all tokens.

Closes #160

### CODE REVIEW

File: custom_components/google_home/manifest.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "domain": "google_home",
   "issue_tracker": "https://github.com/leikoilja/ha-google-home/issues",
   "name": "Google Home",
-  "requirements": ["glocaltokens==0.2.8"],
-  "version": "1.0.1",
+  "requirements": ["glocaltokens==0.3.0"],
+  "version": "1.2.0",
```
Comment (KapJI) [inline custom_components/google_home/manifest.json:11]: This should be updated on new version release.

**Review by leikoilja — APPROVED** [review]
Looks good to me! 

But can we please triple check the integration before we release v1.3.0? 
There are a few updates we did to `glocaltokens` as well as `gpsoauth`, we just need to make sure it's not breaking anything :)

File: custom_components/google_home/manifest.json (hunk 1 above)
Comment (KapJI) [reply custom_components/google_home/manifest.json:11]: Yeah, but I think this should be in a separate PR with github release on top of it.

**Comment by DurgNomis-drol** [comment]
@leikoilja I will test it later today and @KapJI also make sure that `remove_service` is also fixed. 

I am also nearly done with another feature, but haven pushed it yet. This will add a switch for Do Not Disturb mode. 😴 Lets merge this first before I make a PR for that. I don't know if you guys want to wait to release a new version of `google_home` or we just should make it part of the next one.

**Comment by KapJI** [comment]
I did some basic tests, at the first glance looks fine.

I'll wait before @DurgNomis-drol also tests it.

**Comment by KapJI** [comment]
Ok, let's merge this one and wait for testing results before releasing the new version.

**Comment by DurgNomis-drol** [comment]
Just test it and it works! 🚀 Great job guys! We are really on a role at the moment! 😄

**Pull request merged**